src/rust/ide/lib/view/src/graph_editor/component/type_coloring.rs

```diff
@@ -0,0 +1,55 @@
+//! This module contains functionality that allows ports and edges to be colored according
+//! to their type information.
+
+use crate::prelude::*;
+
+use crate::graph_editor::SharedHashMap;
+use crate::graph_editor::Type;
+
+use ensogl::data::color;
+use std::collections::hash_map::DefaultHasher;
+use std::hash::Hasher;
+
+
+/// Color that should be used if no type information is available.
+pub const DEFAULT_TYPE_COLOR : color::Lcha = color::Lcha::new(0.5, 0.0, 0.0, 1.0);
```

Comment:
no type information is not DEFAULT type color. The name is misleading. It should be named MISSING_TYPE_INFO_COLOR instead